packages/react-core/src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Panel/panel';
+import { css } from '@patternfly/react-styles';
+
+export interface PanelProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the panel */
+  children?: React.ReactNode;
+  /** Class to add to outer div */
+  className?: string;
+  /** Styling to apply to the perimeter of the panel */
+  perimeterStyling?: 'raised' | 'bordered';
```

Comment:
There may be other variations that could be used as a value here that aren't necessarily tied to the border/box-shadow (perimeter styling). I wonder if we could use something more generic? `variation` is something we use in other components - could we use that, or is that too generic and/or we might want to reserve that for other types of variations we could introduce to the component?

Reply:
@mcoker how does `variant` sound? I know that's what we use on Button for instance.

Reply:
wfm! But I'll defer to @nicolethoen, @tlabaj, @kmcfaul since they're more familiar with prop naming.